tests/e2e/slashing.go

```diff
@@ -14,249 +14,170 @@ import (
+		valsetUpdateN := providerKeeper.GetValidatorSetUpdateId(s.providerCtx())
 
-	// check that slashing packet gets acknowledged
-	ack := channeltypes.NewResultAcknowledgement([]byte{byte(1)})
-	err = s.path.EndpointA.AcknowledgePacket(packet, ack.Acknowledgement())
-	s.Require().NoError(err)
-}
+		// receive the downtime packet on the provider chain. RecvPacket() calls the provider endblocker twice
+		err = s.path.EndpointB.RecvPacket(packet)
+		s.Require().NoError(err)
 
-func (s *CCVTestSuite) TestSendSlashPacketDoubleSign() {
-	s.SetupCCVChannel()
-	s.SetupTransferChannel()
-	validatorsPerChain := len(s.consumerChain.Vals.Validators)
+		// We've now advanced two blocks.
 
-	providerStakingKeeper := s.providerApp.GetE2eStakingKeeper()
-	providerSlashingKeeper := s.providerApp.GetE2eSlashingKeeper()
-	providerKeeper := s.providerApp.GetProviderKeeper()
-	consumerKeeper := s.consumerApp.GetConsumerKeeper()
+		// One VSC packet should have been sent during block n
+		expectedSentValsetUpdateId := valsetUpdateN
+		_, found = providerKeeper.GetVscSendTimestamp(s.providerCtx(),
+			s.consumerChain.ChainID, expectedSentValsetUpdateId)
+		s.Require().True(found)
 
-	// get a cross-chain validator address, pubkey and balance
-	tmVals := s.consumerChain.Vals.Validators
-	tmVal := tmVals[0]
+		// Confirm the valset update Id was incremented twice.
+		valsetUpdateNPlus2 := providerKeeper.GetValidatorSetUpdateId(s.providerCtx())
+		s.Require().Equal(valsetUpdateN+2, valsetUpdateNPlus2)
 
-	val, err := tmVal.ToProto()
-	s.Require().NoError(err)
-	pubkey, err := cryptocodec.FromTmProtoPublicKey(val.GetPubKey())
-	s.Require().Nil(err)
-	consAddr := sdk.GetConsAddress(pubkey)
-	valData, found := providerStakingKeeper.GetValidatorByConsAddr(s.providerCtx(), consAddr)
-	s.Require().True(found)
-	valOldBalance := valData.Tokens
-
-	// create the validator's signing info record to allow jailing
-	valInfo := slashingtypes.NewValidatorSigningInfo(consAddr, s.providerCtx().BlockHeight(),
-		s.providerCtx().BlockHeight()-1, time.Time{}.UTC(), false, int64(0))
-	providerSlashingKeeper.SetValidatorSigningInfo(s.providerCtx(), consAddr, valInfo)
-
-	// get valseUpdateId for current block height
-	valsetUpdateId := consumerKeeper.GetHeightValsetUpdateID(s.consumerCtx(), uint64(s.consumerCtx().BlockHeight()))
-
-	// construct the downtime packet with the validator address and power along
-	// with the slashing and jailing parameters
-	validator := abci.Validator{
-		Address: tmVal.Address,
-		Power:   tmVal.VotingPower,
-	}
+		// check that the validator was removed from the provider validator set
+		s.Require().Len(s.providerChain.Vals.Validators, validatorsPerChain-1)
 
-	oldBlockTime := s.consumerCtx().BlockTime()
-	packetData := types.NewSlashPacketData(validator, valsetUpdateId, stakingtypes.DoubleSign)
+		// Relay the VSC packet to the consumer
+		relayAllCommittedPackets(s, s.providerChain, s.path, ccv.ProviderPortID, s.path.EndpointB.ChannelID, 1)
```

Comment:
This is the new way that we relay the VSC packet from provider to consumer. Note that this method is commonly used elsewhere

Reply:
I've also updated the comment for `TestRelayAndApplySlashPacket` to make this more clear